preview-maker.js

```diff
@@ -129,44 +129,34 @@ const addWallpaper = async (elements, wallpaper) => {
   return result;
 };
 
-function calculateAccentColor(colors) {
-  let colorsQuantity = new Map();
-  let max = 0;
-  let accentHue;
-
+const accentColor = (colors, grey) => {
+  const quantity = new Map();
+  let accentQuantity = 0;
+  let accent;
   for (const color of colors) {
-    colorsQuantity.set(color, (colorsQuantity.get(color) ?? 0) + 1);
-
-    if (colorsQuantity.get(color) > max) {
-      accentHue = color;
-      max = colorsQuantity.get(color);
+    quantity.set(color, (quantity.get(color) ?? 0) + 1);
+    if (quantity.get(color) > accentQuantity) {
+      accent = {
```

Comment:
it's just better to use two variables for the background color and the shadow color and construct an object when returning.